examples/systemcalls/systemcalls.c

```diff
@@ -93,6 +146,49 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
  *
 */
 
+    int status;
```

Comment:
@charan1811 how could you share logic with the function above and avoid duplication?